custom/icds_reports/reports/incentive.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import unicode_literals, absolute_import
+
+import pytz
+
+from custom.icds_reports.models.aggregate import AWWIncentiveReport
+from custom.icds_reports.utils import india_timezone
+
+class IncentiveReport(object):
+
+    def __init__(self, block, month):
+        self.block = block
+        self.month = month
+
+    @property
+    def india_now(self):
```

Comment:
any reason this isn't a utility elsewhere? also if it should return the same for the class, then it should be cached